src/guide/essentials/reactivity-fundamentals.md

```diff
@@ -212,6 +271,66 @@ En el ejemplo anterior, se llamará al método `increment` cuando se haga clic e
 
 </div>
 
+### Reactividad profunda {#deep-reactivity}
+
+<div class="options-api">
+
+En Vue, el estado es profundamente reactivo por defecto. Esto significa que puedes esperar que se detecten cambios incluso cuando mutas objetos o arrays anidados:
+
+```js
+export default {
+  data() {
+    return {
+      obj: {
+        nested: { count: 0 },
+        arr: ['foo', 'bar']
+      }
+    }
+  },
+  methods: {
+    mutateDeeply() {
+      // estos funcionarán como se esperaba.
+      this.obj.nested.count++
+      this.obj.arr.push('baz')
+    }
+  }
+}
+```
+
+</div>
+
+<div class="composition-api">
+
+Las refs pueden tener cualquier tipo de valor, incluyendo objetos profundamente anidados, arrays, o estructuras que son parte de JavaScript como `Map`.
+
+Una ref hará que su valor sea profundamente reactivo. Esto significa que puedes esperar que se detecten cambios aún cuando mutas objetos o arrays anidados.
+
+```js
+import { ref } from 'vue'
+
+const obj = ref({
+  nested: { count : 0 },
+  arr: ['foo', 'bar']
+})
+
+function mutateDeeply() {
+  // estos funcionarán como se esperaba.
+  obj.value.nested.count++
+  obj.value.arr.push('baz')
+}
+```
+
+Valores no primitivos son convertidos en proxies reactivas mediante [`reactive()`](#reactive), lo que es discutido más abajo.
+
+También es posible no optar por reactividad profunda con [refs superficiales](/api/reactivity-advanced#shallowref). Para refs superficiales, solo el acceso de `.value` es seguido para reactividad. Las refs superficiales pueden ser usadas para optimizar rendimiento al evitar los costos de observación de objetos grandes, o en casos donde el estado interno es manejado por una librería externa.
```

Comment:
"sólo se registra el acceso a `.value` para la reactividad." instead of "solo el acceso de `.value` es seguido para reactividad"